Makefile

```diff
@@ -73,7 +73,9 @@ test_fast::
 	./sdk/nodejs/node_modules/mocha/bin/mocha ./sdk/nodejs/bin/tests
 
 test_all:: test_fast
-	PATH=$(PULUMI_BIN):$(PATH) $(GO) test -v -count=1 -cover -timeout 1h -parallel ${TESTPARALLELISM} $(TESTABLE_PKGS)
+	if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then \
+		PATH=$(PULUMI_BIN):$(PATH) $(GO) test -v -count=1 -cover -timeout 1h -parallel ${TESTPARALLELISM} $(TESTABLE_PKGS) \
+	fi
```

Comment:
Won’t this also prevent local dev use of `make test_all`?  In general, feels like we shouldn’t have Travis specific things in the Makefile if possible.

Reply:
@lukehoban this is a WIP, I didn't think anyone would be online.